yb-voyager/cmd/exportData.go

```diff
@@ -213,19 +213,42 @@ func extractTableListFromString(flagTableList string) []string {
 		return []string{}
 	}
 	tableList := utils.CsvStringToSlice(flagTableList)
-	if source.DBType != POSTGRESQL {
+	if source.DBType == MYSQL {
 		return tableList
 	}
 	// in postgres format should be schema.table, public is default and other parts of code assume schema.table format
 	for _, table := range tableList {
 		parts := strings.Split(table, ".")
+		if len(parts) > 2 {
+			utils.ErrExit("invalid table name: %q in the --table-list flag.", table)
+		}
+		var tableName string
 		if len(parts) == 1 {
-			finalTableList = append(finalTableList, "public."+table)
-		} else if len(parts) == 2 {
-			finalTableList = append(finalTableList, table)
+			tableName = parts[0]
+		} else {
+			tableName = parts[1]
+		}
+		if utils.IsQuotedString(tableName) {
+			if  source.DBType == ORACLE {
+				tableName = strings.Trim(tableName,`"`)
+			}
```

Comment:
For PG and MySQL, won't you need to trim the double quotes?

Reply:
No, for PG we need not to remove the double quotes, the queries work with that as is. And for MySQL as the table-name are case-Sensitive the quotes are not even there and also I had tried creating a table with quotes but it seems like that it doesn't happen in MySQL - 
```
MySQL [pk_missing]> create table "Test"(id int);
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '"Test"(id int)' at line 1
```
so we need not to handle anything for MySQL